js/filter-panel.js

```diff
@@ -25,21 +28,25 @@ function FilterPanel(options) {
   this.$toggle.on('click', this.toggle.bind(this));
 
   this.filterSet = new FilterSet(this.$form).activate();
-  if (!_.isEmpty(this.filterSet.serialize())) {
-    this.show();
-  }
 
   this.adjust();
 }
 
 FilterPanel.prototype.adjust = function() {
```

Comment:
We can probably come up with a clearer name for this method--maybe something like `setInitialDisplay`. I think the behavior is what @noahmanger intended--we want the panel to default to open on big screens and closed on small screens, with no intermediate widths.